src/configValidation.js

```diff
@@ -11,15 +12,32 @@ const schema = Joi.object().keys({
   },
   esclient: Joi.object().keys({
     requestTimeout: Joi.number().integer().min(0)
-  }).unknown(true)
-}).requiredKeys('dbclient', 'dbclient.statFrequency', 'esclient').unknown(true);
+  }).unknown(true),
+  schema: Joi.object().keys({
+    indexName: Joi.string()
+  })
+}).requiredKeys('dbclient', 'dbclient.statFrequency', 'esclient', 'schema.indexName').unknown(true);
 
 module.exports = {
   validate: function validate(config) {
     Joi.validate(config, schema, (err, value) => {
       if (err) {
         throw new Error(err.details[0].message);
       }
+
+      // now verify that the index exists
+      const esclient = new elasticsearch.Client(config.esclient);
+
+      // callback that throws an error if the index doesn't exist
+      const existsCallback = (exists) => {
+        if (!exists) {
+          throw new Error(`elasticsearch index ${config.schema.indexName} does not exist`);
```

Comment:
nice. we should probably expand this error quite a bit and include both links to https://github.com/pelias/schema/ and the [installation docs](http://pelias.io/install.html)

Maybe something like this:

```
console.error(`ERROR: Elasticsearch index ${config.schema.indexName} does not exist`);
console.error('You must use the pelias-schema tool (https://github.com/pelias/schema/) to create the index first');
console.error('For full instructions on setting up Pelias, see http://pelias.io/install.html');
```